### DISCUSSION TITLE
Creating a simple static, local road map

### DISCUSSION THREAD

**Discussion by louhy** [mapsforge/mapsforge#1302] [Q&A] [answered]
Sorry if this is a stupid question but I'd really just like a simple static map of a city-metro sized area, including roads, that can be shown in a small Android app. It looks like there's a map-creator script that I guess can create this, but it looks like it takes quite a bit of setup (and even then I'm not sure if it's going to give me what I need). 

I actually tried using an entire US northeast map (although I'm not even sure that includes roads?) but it's large enough that I don't think it's very mobile friendly, opening up such a large file. Am I going down the right path for a simple use case like this or is there something easier I can do? 

To summarize the goal I have in mind, I'd just like to be able to bring up a location marker on a map of the local area, given specific GPS coordinates from the phone's location. Preferably without pulling any data from the Internet. Is that a good use-case for mapsforge?

**Comment by devemux86** [comment, marked as answer]
If you ask how to create offline map files, you can see the [map creation guide](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md).

[mapsforge-creator](https://github.com/mapsforge/mapsforge-creator) automatically generates Mapsforge maps using Geofabrik
and it is based on the above guide.

---

You can also check [VTM](https://github.com/mapsforge/vtm) OpenGL map library (compatible with Mapsforge).

**Reply by louhy** [reply]
I guess what's confusing me is: 
-The instructions don't seem to be about road maps (just land maps).
-Because of all the dependencies needed, the automated process looks just as complicated as the creation guide

So I guess you'd recommend trying this automated process, installing this "Osmosis" thing etc?

**Comment by devemux86** [comment]
You can filter what is written in the map with [map-writer](https://github.com/mapsforge/mapsforge/blob/master/docs/Getting-Started-Map-Writer.md) tag-mapping
and what appears on the screen with the [render themes](https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md).
